src/client/components/app/ScanNetworkIcon/index.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Address, Network } from '@types';
+import { DefaultscanIcon, EtherscanIcon, FtmscanIcon, Icon } from '@components/common';
+
+interface ScanNetworkIconProps {
+  currentNetwork?: Network;
+  blockExplorerUrl?: string;
+  address: Address;
+}
+
+const IconScan = styled(Icon)`
+  fill: black;
```

Comment:
Here we can use a color based on the theme, so that we have better contrast when using a darker theme.
![image](https://user-images.githubusercontent.com/18649057/140393653-d46ccc44-f6df-4465-9316-9ffd17c763a1.png)
Try using 
`fill: ${({ theme }) => theme.colors.onPrimaryVariant}`
and check if contrast looks good on all themes

Reply:
- Etherscan logo looks good with all themes
- Fantomscan logo is hard to see on Mono Dark and Classic v2 
![image](https://user-images.githubusercontent.com/92565284/140434320-1547fe15-3257-47e4-984f-b69449b34f0d.png)
![image](https://user-images.githubusercontent.com/92565284/140434368-74886abc-96c0-4d9b-95ba-971d81514b9f.png)

Reply:
I will update the var so I check every case then!

Reply:
Asked the designer for a new version of the SVG, since the stokes of the cube where not transparent, it was not handled well in themes @TacosTonight pointed.